src/iop/highlights.c

```diff
@@ -2172,9 +2172,10 @@ void gui_focus(struct dt_iop_module_t *self, gboolean in)
   dt_iop_highlights_gui_data_t *g = (dt_iop_highlights_gui_data_t *)self->gui_data;
   if(!in)
   {
+    const gboolean was_visualize = dt_bauhaus_widget_get_quad_active(g->clip);
```

Comment:
same here.

Or is there some subtle difference that I missed?

Reply:
same comment as above